fairlearn/postprocessing/_threshold_optimizer.py

```diff
@@ -62,15 +62,12 @@ class ThresholdOptimizer(ClassifierMixin, BaseEstimator):
     :type grid_size: int
     :param flip: Allow flipping to negative weights if it improves accuracy.
     :type flip: bool
-    :param plot: Show ROC/selection-error plot if True
-    :type plot: bool
     """
 
     def __init__(self, *, unconstrained_predictor=None, estimator=None,
-                 constraints=DEMOGRAPHIC_PARITY, grid_size=1000, flip=True, plot=False):
+                 constraints=DEMOGRAPHIC_PARITY, grid_size=1000, flip=True):
         self.grid_size = grid_size
         self.flip = flip
-        self.plot = plot
         self.post_processed_predictor_by_sensitive_feature = None
```

Comment:
this should be moved to `fit` and renamed to `self.postprocessed_predictor_by_sensitive_feature_` (trailing underscore)

Reply:
I'll have another PR for this since I finally got everything passing. Thanks for spotting it!